docs/batch-changes/batch-spec-yaml-reference.mdx

```diff
@@ -2,6 +2,28 @@
 
 <p className="subtitle">Sourcegraph Batch Changes use [batch specs](/batch-changes/create-a-batch-change#writing-a-batch-spec) to define batch changes. This page is a reference guide to the batch spec YAML format in which batch specs are defined.</p>
 
+## `version`
+
+The `version` of the batch spec. Defaults to 1 if not specified. New batch specs should use version 2.
+We recommend to always specify the version.
+
+### Version 1
+
+default
```

Comment:
I find this a bit confusing, because we actually want users to consider `Version 2` the default choice. Maybe we could say "The schema version before Sourcegraph version 5.5. For now, if no version is specified, batch changes will use this schema version. It is recommended to switch to version 2."